Dockerfile

```diff
@@ -13,8 +13,15 @@ LABEL name="rhosp16.1/openstack-cinder-volume-hpe" \
 # switch to root and install a custom RPM, etc.
 USER "root"
 
+
+RUN export http_proxy=http://web-proxy.atl.hpecorp.net:8080/ && export https_proxy=http://web-proxy.atl.hpecorp.net:8080/
```

Comment:
Shall we remove the proxy from Dockerfile?

Reply:
done